test_scripts/AppServices/commonAppServices.lua

```diff
@@ -146,7 +145,6 @@ function commonAppServices.getAppServiceProducerConfig(app_id, service_type)
     priority = "NONE",
     default_hmi = "NONE",
     groups = { "Base-4" , "AppServiceProvider" },
-    nicknames = { config["application" .. app_id].registerAppInterfaceParams.appName },
```

Comment:
Why was this field removed?

Reply:
@jacobkeeler 
Sometimes we run into the issue when app is unable to register due to nickname validation.
Let me try to reproduce it and collect the logs.

Reply:
@jacobkeeler Issue is not reproduced anymore using latest SDL and Scripts. 
Definition of nicknames reverted back in d858303